module/data/item/templates/physical-item.mjs

```diff
@@ -35,6 +35,20 @@ export default class PhysicalItemTemplate extends SystemDataModel {
     };
   }
 
+  /* -------------------------------------------- */
+  /*  Getters                                     */
+  /* -------------------------------------------- */
+
+  /**
+   * Get a human-readable label for the price and denomination.
+   * @type {string}
+   */
+  get priceLabel() {
+    const { value, denomination } = this.price;
+    const hasPrice = value && (denomination in CONFIG.DND5E.currencies);
+    return hasPrice ? `${value} ${game.i18n.localize(CONFIG.DND5E.currencies[denomination].label)}` : null;
```

Comment:
`CONFIG.DND5E.currencies.x.label` should already be localised by this point I believe.
```suggestion
    return hasPrice ? `${value} ${CONFIG.DND5E.currencies[denomination].label}` : null;
```